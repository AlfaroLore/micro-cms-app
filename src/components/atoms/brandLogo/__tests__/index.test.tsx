import { render } from '@testing-library/react';
import BrandLogo from '..';

describe('BrandLogo', () => {
  it('Renders BrandLogo', () => {
    const component = render(<BrandLogo />);

    expect(component).toBeTruthy();
    expect(component.asFragment()).toMatchSnapshot();
  });
});
