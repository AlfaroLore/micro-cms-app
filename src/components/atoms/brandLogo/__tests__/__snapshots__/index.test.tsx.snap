// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BrandLogo > Renders BrandLogo 1`] = `
<DocumentFragment>
  <img
    alt="Logo"
    src="/src/assets/react.svg"
  />
</DocumentFragment>
`;
